Assignment_0/assignment_0_210551.md

```diff
@@ -0,0 +1,86 @@
+﻿#HACK\_It
+
+` `Assignment\_0
+
+Name- KUSHAGRA GUPTA
+Roll No- 210551
+
+#Unix File-System
+
+
+
+`   `Commands-
+
+
+
+1. pwd : Prints current Directory 
+1. rmdir : removes the directory, specified by the Directory parameter, from the system
+1. cd : The cd command can be used to either change to a directory that is relative to the the location of the current working directory or to an absolute location in the filesystem.
+1. heirarchy up , **"cd** takes you to root directory.
+1. ls : prints all directories and files present in current directories. 
+
+`            `**ls -l**   - lists all content with detailed info 
+
+`            `**ls -a**  -lists hidden files as well 
+
+`            `**ls -h**  -show memory of files in more readable format.
+
+1. touch : creates a new file in current directory.
+1. rm : remove file
+1. cat : shows content of file.
+1. man : used to display the user manual of any command that we can run on the terminal.
+1. echo : echo *text* prints *text* on the terminal(**displaying lines of text or string which are passed as arguments on the command line)**
+1. chmod : change mode. Used to change user permissions.
+1. grep : globally search a regular expression. Used to search a file for a regular expression
+1. sudo : allows you to run programs with the security privileges of another user .
+1. su - used to switch to another user
+1. mv- rename a file 
+1. rm - u**sed to remove objects such as computer files, directories and symbolic links from file systems and also special files**.
+1. sudo su - **allows you to run programs as another user, by default the root user**
+1. cd /mnt/ -used to access windows using WSL
+1. history -used to see older commands
+1. history > -c to delete history
+1. hostname - determine system
+1. VIM -used to edit text files
+
+
+#VIM
+
+Commands-
+
+
+
+\1) h, k,l,j - to move left,up,right,down respectively

```

Comment:
backslash not needed. Also opt for non-weighted lists, i.e. unordered lists in such cases cause there's no priority.